content/en/network_monitoring/devices/network_topology_map.md

```diff
@@ -15,62 +15,209 @@ further_reading:
+2. Datadog Agent version 7.52 or later is installed.
+
+## Navigation options
+
+In the Network Topology Map, the following navigation options are available:
+
+### View
+
+1. Under **View By**, select to visualize devices in groups according to tags: 
+
+{{< img src="/network_device_monitoring/network_topology_map/network_view_by_location_2.png" alt="The navigation option, with view by devices and tags selected, highlighting view by location" style="width:80%;" >}}
+
+### Color
+
+2. Under **Color By**, change how nodes on the Device Topology Map are colored based on:
+
+- **Device State**: Display nodes on the Device Topology Map by SNMP reachability.
+- **Ping State**: Display nodes on the Device Topology Map by Ping status. 
+
+{{< img src="/network_device_monitoring/network_topology_map/device_topology_color_by_device.png" alt="The navigation option, with view by color selected, highlighting view by device state" style="width:80%;" >}}
+
+   The following are the definitions of the nodes for each color state:
+   <div style="width:80%; margin: 0 auto;">
+
+   | Color    | Description               |
+   |----------|---------------------------|
+   | Green   | Device is reachable.      |
+   | Red   | Issue with device, such as unreachable through SNMP.  |
+   | Gray    | The device is monitored by NDM, however, no data has been received. For example, if the ping wasn't configured and you opted to `color by` **Ping State** in the Device Topology Map, the devices are displayed in gray. |
+   | No color | Shadow devices that are not directly monitored by NDM, but are discoverable through LLDP/CDP from a connected device that NDM is monitoring. You can toggle on/off the [Hide _N_ unmonitored device section](#filter-devices) if you want these devices to be shown on the Device Topology Map.         |
+
+   </div>
+
+### Filter devices
+
+3. Under **Filter Devices**, gain further granular control over what devices are shown on the Device Topology Map. 
+
+{{< img src="/network_device_monitoring/network_topology_map/device_topology_filter_devices_hide.png" alt="The navigation option, with view filer option selected, toggling by hide unmonitored devices " style="width:80%;" >}}
+
+**Note:** The filter devices setting impacts what devices are shown on the Device Topology Map for _all_ queries you may make, such as if you filter by a device facet in the search bar. 
+
+Hide _N_ Unmonitored Devices - Turned OFF by default.
+: Toggling this on hides devices on the Device Topology Map that are not directly monitored by Network Device Monitoring, but still discovered by LLDP/CDP, and shown on the map from adjacent devices that are monitored by Network Device Monitoring.
+
+Hide _N_ Unconnected Devices - Turned ON by default.
+: Toggling this on hides any devices that have no link connections. Devices can be unconnected for reasons such as improper configuration, or the device does not support [LLDP/CDP](#troubleshooting).
+
+### Icon legend 
```

Comment:
this section is under **Navigation options**. Is it an option they can control? If not, consider moving it.

Reply:
Yes it's a toggle, thanks for the tip though!